service/src/users-api.js

```diff
@@ -14,6 +18,7 @@ const {
   jwksUri,
   issuer
 } = require('./config');
+const { fromJson } = require('@bufbuild/protobuf');
```

Comment:
would be great to either hide this in the API, or at the very least make the import from @aserto-node.